plugins/dac_data_manager.c

```diff
@@ -826,18 +826,16 @@ static GtkWidget *spin_button_create(double min, double max, double step, unsign
 static GtkWidget *frame_with_table_create(const char *frm_title, unsigned rows, unsigned columns)
 {
 	GtkWidget *frame;
-	GtkWidget *align;
+	// GtkWidget *box;
```

Comment:
What the plan with these commented lines? Can you leave a to-do next to the code?

Reply:
added suggestive TO DO comments